docs/components/kerbside.md

```diff
@@ -0,0 +1,83 @@
+# Kerbside
+
+Kerbside is a [SPICE](https://spice-space.org/) protocol native VDI proxy
+responsible for providing rich VDI experiences to users of Shaken Fist, oVirt,
+OpenStack, or any other cloud where the hypervisor is capable of providing SPICE
+consoles over the network.
+
+The SPICE protocol has existed for a long time ago, and still represents the
+richest and most performant option for remote desktops using Open Source
+technologies. However before Kerbside, SPICE adoption was limited because the
+console was generally provided in the form of a HTML5 transcoded interface in
+a web client. Unfortunately these HTML5 interfaces do not support many of the
+more novel features of the SPICE protocol, nor do they support high resolution
+desktops. By avoiding transcoding to a HTML5 client, we avoid these problems.
```

Comment:
Alternative lower verbosity version:

Before Kerbside, consoles were generally provided by a HTML5 transcoded interface in a web client. Unfortunately, HTML5 interfaces do not support many of the more novel features of the SPICE protocol, nor do they support high resolution desktops. Kerbside avoids these problems, and brings SPICE to the masses.